packages/workbox-cli/src/lib/errors.js

```diff
@@ -24,4 +24,14 @@ module.exports = {
   'config-validation-failed': `Your configuration is invalid:`,
   'workbox-build-runtime-error': `Service worker generation failed:`,
   'unknown-command': `Unknown command:`,
+  'no-file-extensions-found': ol`No files could be found that are suitable for
+    caching.`,
+  'no-file-extensions-selected': `Please select at least one file extension.`,
+  'sw-src-does-not-exist': ol`The source service worker file you provided does
+    not exist. Please provide the path to an existing file.`,
+  'invalid-sw-dest': ol`Please enter a valid path to use for the service worker
+    file that's created.`,
+  'glob-directory-invalid': ol`The path you entered isn't a valid directory.`,
+  'invalid-config-location': ol`Please enter a valid path to use for the saved
```

Comment:
Does this require a file structure to exist to work? or does this error occur in other situations?

Reply:
Similar to the `invalid-sw-dest` error, it's something that gets triggered when you enter an empty string.

Also similar, if it's a path that's not valid, you'll end up with a runtime error when the write operation fails.